agent/firewall_test.go

```diff
@@ -119,11 +119,11 @@ func TestCreateRules(t *testing.T) {
 	fw.CreateRules(0)
 
 	// expect
```

Comment:
That comment doesn't add anything, since the variable is already called "expect".